### PR TITLE
SEO-172143 page with redirect links

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @MallikaRK,
Please review changes made
Regards,
Scholar.

### CODE REVIEW

File: WindowsForms/Gradient-Label/Getting-Started.md [hunk 1]
```diff
@@ -116,4 +116,4 @@ Me.gradientLabel1.ForeColor = System.Drawing.Color.SeaShell
 
 {% endtabs %}
 
-![Windows Forms GradientLabel shows grdient style](GradientLabel-Images\gradientlabel_style.jpeg)
+![Windows Forms GradientLabel shows grdient style](gradientlabel-images/windowsforms-gradientlabel_style.jpeg)
```
Comment (MallikaRK) [inline WindowsForms/Gradient-Label/Getting-Started.md:119]: give hypen in between gradientlabel.
gradient-label-style
[Use the image name in same format, hypen[-], undescrore[_] both are used here. Use hypen symbol for image name]
Remove _ [underscore] and use Hypen for all places.

File: WindowsForms/Gradient-Panel/Getting-Started.md [hunk 2]
```diff
@@ -25,11 +25,11 @@ Click [NuGet Packages](https://help.syncfusion.com/windowsforms/installation/ins
 
     * Syncfusion.Shared.Base
 
-![Windows Forms GradientPanel drag and drop from toolbox](GradientPanel-Images/GradientPanel_toolbox.png)
+![Windows Forms GradientPanel drag and drop from toolbox](gradientpanel-images/windowsforms-gradientpanel_toolbox.png)
```
Comment (MallikaRK) [inline WindowsForms/Gradient-Panel/Getting-Started.md:28]: recheck

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/Gradient-Label/Getting-Started.md (hunk 1 above)
Comment (Scholar-jaruha) [reply WindowsForms/Gradient-Label/Getting-Started.md]: changed

File: WindowsForms/Gradient-Panel/Getting-Started.md (hunk 2 above)
Comment (Scholar-jaruha) [reply WindowsForms/Gradient-Panel/Getting-Started.md]: changed

**Review by MallikaRK — CHANGES_REQUESTED** [review]

File: WindowsForms/Gradient-Label/Getting-Started.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ The following steps describe how to create a **GradientLabel** control via desig
     * Syncfusion.Tools.Base
     * Syncfusion.Tools.Windows
 
-![Drag and drop GradientLabel from toolbox](GradientLabel-Images/Overview_img600.jpeg) 
+![Drag and drop GradientLabel from toolbox](gradient-label-images/windowsforms-gradient-label-toolbox.jpeg)
```
[the remark this answers is not in the document]
Comment (Scholar-jaruha) [reply WindowsForms/Gradient-Label/Getting-Started.md:35]: Changed

File: WindowsForms/gradientpanelext/Getting-Started.md [hunk 4]
```diff
@@ -27,13 +27,13 @@ The following steps describe how to create the **GradientPanelExt** control via
 
     * Syncfusion.Shared.Base
 
-![Drag and drop GradientPanelExt from toolbox](GradientPanelExt_images/Overview_img373.png)
+![Drag and drop Gradient Panel from toolbox](gradient-panel_images/windowsforms-gradient-panel-toolbox.png)
 
 3. Set the Gradient colors to the panel by defining gradient styles and colors.
 
-4. Add the primitives to the control using **GradientPanelExt PrimitiveCollection Editor**, which is displayed using the [Primitives](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Tools.GradientPanelExt.html#Syncfusion_Windows_Forms_Tools_GradientPanelExt_Primitives) property. Refer to the following screenshot.
+4. Add the primitives to the control using **GradientPanelExt PrimitiveCollection Editor**, which is displayed using the [Primitives](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Tools.GradientPanelExt.html#Syncfusion_Windows_Forms_Tools_gradient-panel_Primitives) property. Refer to the following screenshot.
```
Comment (MallikaRK) [inline WindowsForms/gradientpanelext/Getting-Started.md:34]: Update the valid URL here. This URL seems wrong.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/Gradient-Label/Getting-Started.md [hunk 5]
```diff
@@ -116,4 +116,4 @@ Me.gradientLabel1.ForeColor = System.Drawing.Color.SeaShell
 
 {% endtabs %}
 
-![Windows Forms GradientLabel shows grdient style](GradientLabel-Images\gradientlabel_style.jpeg)
+![Windows Forms GradientLabel shows grdient style](gradient-label-images/windowsforms-gradient-label_style.jpeg)
```
Comment (MallikaRK) [inline WindowsForms/Gradient-Label/Getting-Started.md:119]: Already said, don't use underscore in image name. 
windowsforms-gradient-label**_**style.jpeg - wrong

windowsforms-gradient-label**-**style.jpeg
Use hyphen for image names.
Check and update it.

File: WindowsForms/gradientpanelext/Getting-Started.md (hunk 4 above)
Comment (MallikaRK) [reply WindowsForms/gradientpanelext/Getting-Started.md:34]: This is the valid link
https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Tools.GradientPanelExt.html#Syncfusion_Windows_Forms_Tools_GradientPanelExt_Primitives

please check and update valid links in further your works.

File: WindowsForms/gradientpanelext/Getting-Started.md [hunk 6]
```diff
@@ -27,13 +27,13 @@ The following steps describe how to create the **GradientPanelExt** control via
...
 
-![Windows Forms GradientPanelExt showing primitive collection editor](GradientPanelExt_images/Overview_img374.png)
+![WindowsForms Gradient Panel showing primitive collection editor](gradient-panel_images/windowsforms-gradient-panel-collection-editor.png)
```
Comment (MallikaRK) [inline WindowsForms/gradientpanelext/Getting-Started.md:36]: Which one is correct image folder name?
image
Use the proper image folder name.

File: WindowsForms/gradientpanelext/Getting-Started.md [hunk 7]
```diff
@@ -246,4 +246,4 @@ gradientPanelExt.Controls.Add(textBoxExt2)
 {% endcapture %}
 {{ codesnippet3 | OrderList_Indent_Level_1 }}
 
-![Windows Forms GradientPanelExt shows added primitives and other controls](GradientPanelExt_images\GradientPanelExt_addcontrols.png)
+![WindowsForms Gradient Panel shows added primitives and other controls](gradient-panel_images\windowsforms-gradient-panel-addcontrols.png)
```
Comment (MallikaRK) [inline WindowsForms/gradientpanelext/Getting-Started.md:249]: Check and change the Image Folder name in all places.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/gradientpanelext/Getting-Started.md (hunk 6 above)
Comment (Scholar-jaruha) [reply WindowsForms/gradientpanelext/Getting-Started.md:36]: Issue Fixed

File: WindowsForms/gradientpanelext/Getting-Started.md (hunk 7 above)
Comment (Scholar-jaruha) [reply WindowsForms/gradientpanelext/Getting-Started.md]: Issue resolved

File: WindowsForms/Gradient-Label/Getting-Started.md (hunk 5 above)
Comment (Scholar-jaruha) [reply WindowsForms/Gradient-Label/Getting-Started.md]: Issue resolved

File: WindowsForms/gradientpanelext/Getting-Started.md (hunk 4 above)
Comment (Scholar-jaruha) [reply WindowsForms/gradientpanelext/Getting-Started.md]: Issue resolved

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**